src/controllers/request-controller.ts

```diff
@@ -88,11 +90,14 @@ export default class RequestController {
         const body = await this.#requestPresentation.body(request)
         return res.status(StatusCodes.ACCEPTED).json(body)
       } else {
+        await this.ceramicService.pinStream(streamId)
```

Comment:
I don't know what's going on with the tests.  If I comment out this line, the tests pass consistently.  They also pass consistently when I run them from Webstorm.  But when running ceramic_integration.test.ts from a terminal or in CircleCI (and this line isn't commented out), it's consistently hanging while tearing down ceramic and/or ipfs, and timing out.  Driving me kind of crazy tbh.  I'm considering wrapping the test teardown in a try/catch that swallows all errors, but I'm open to other suggestions!

Reply:
okay I figured it out.  Adding pinStream here makes creating the request take longer, which meant that when the anchor runs, the request might not even exist yet in the CAS database.  Putting a sleep here has the same effect.

I increased the time we wait before starting an anchor to stabilize the test, and filed https://github.com/ceramicnetwork/js-ceramic/issues/1919 which would make sleeping in the test unnecessary.

Reply:
Ohh, you sure about this? What was the motivation behind this? This would make a request take longer. Can we do this off-request maybe?